src/constants.js

```diff
@@ -15,5 +15,26 @@ export const URL = url.parse(
         : 'https://collector.thundra.io/api'
 );
 
+export class TimeoutError extends Error {
+    constructor(message) {
+        super();
+        this.message = message;
+        this.name = 'TimeoutError';
+        Error.captureStackTrace(this, TimeoutError);
+        Object.setPrototypeOf(this, TimeoutError.prototype);
```

Comment:
just for curiosity, why setting prototype is needed here?

Reply:
To pass the tests. During the transpilation when running tests `extends` keyword did not work. So I added setPrototypeOf to ensure inheritance. Tests passed after `setPrototypeOf`.